.github/workflows/check_metadata_pr.yml

```diff
@@ -0,0 +1,36 @@
+name: Check PR
+
+# Controls when the workflow will run
+on:
+  pull_request_target:
+    branches:
+      - main
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  labeler:
+    runs-on: ubuntu-latest
+    name: Label the PR size
+    steps:
+      - uses: codelytv/pr-size-labeler@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          xs_label: 'size/xs'
+          xs_max_size: '10'
+          s_label: 'size/s'
+          s_max_size: '100'
+          m_label: 'size/m'
+          m_max_size: '200'
+          l_label: 'size/l'
+          l_max_size: '500'
```

Comment:
@pasqualespica what values do we use?

Reply:
according to out first and always true [handbook](https://pagopa.github.io/io-docs/io-handbook/development-workflow#code-reviews) i think  the following one could be a good start : 
```
Each PR should be as small as possible: the general guideline is to create pull requests ranging from 200 to 400 lines of code, not more.

```